lists/pancakeswap-extended.json

```diff
@@ -1332,6 +1332,14 @@
       "decimals": 18,
       "logoURI": "https://tokens.pancakeswap.finance/images/0xa2B726B1145A4773F68593CF171187d8EBe4d495.png"
     },
+    {
+      "name": "Inspect",
```

Comment:
`INSPECT`. Let's just follow the contract
https://zkevm.polygonscan.com/address/0x39aCc7cf02af19A1eB0e3628bA0F5C48f44beBF3#readContract